packages/cactus-plugin-keychain-vault/src/main/json/openapi.json

```diff
@@ -92,23 +92,23 @@
         "summary": "Retrieves the contents of a keychain entry from the backend.",
         "parameters": [],
         "requestBody": {
-          "$ref": "https://raw.githubusercontent.com/hyperledger/cactus/v0.5.0/packages/cactus-core-api/src/main/json/openapi.json#/components/requestBodies/keychain_get_entry_request_body"
+          "$ref": "../../../../cactus-core-api/src/main/json/openapi.json#/components/requestBodies/keychain_get_entry_request_body"
```

Comment:
@Leeyoungone Did you change it to relative paths for a specific reason or just by accident? If for a reason then we might need to break this PR into two, depending on the reason.

Reply:
I think this was a mistake on my end, I've reverted to its original $ref, thank you!